cli/src/main/java/org/mobilitydata/gtfsvalidator/cli/Arguments.java

```diff
@@ -52,6 +54,14 @@ public class Arguments {
               + "It must be a two-letter country code (ISO 3166-1 alpha-2)")
   private String countryCode;
 
+  @Parameter(
+      names = {"-d", "--date"},
+      description =
+          "Date to simulate when validating, in ISO_LOCAL_DATE format like "
+              + "'2001-01-30'. By default, the current date is used. "
+              + "This option enables debugging rules like feed expiration.")
```

Comment:
@bradyhunsaker It's a bit late, but I was wondering about this sentence. I don't see where these debugging rules are enabled.

Reply:
It's not clear. I meant that the main purpose of this option is debugging. For example, it could be used to debug an error with a rule like feed expiration.

There are not special debugging rules associated with it.
I think the use of the word "enables" was a bad choice. Feel free to remove or edit the description.